tests/test_config_reader.py

```diff
@@ -20,3 +20,19 @@ def test_parse_config_files():
         assert pytest_exit_object.type == SystemExit 
         assert pytest_exit_object.value.code == 42  
         assert pytest_exit_object.sys.stdout.getline().strip() == "No file named config.yml found in config directory {}. Exiting...".format(THIS_DIR)
+    
+@pytest.mark.parametrize(
```

Comment:
fantastic!

Now can we add more pairs of inputs and expecteds to test different things?  For example, does it pass if my filenames are "my_data", so add to the list in paramaterize:
``({"general_parameters": {"data_output_filename": "my_data", "data_output_path": 2.2}}, "my_data", "2.2")``
also, more realistic paths so also add to the list:
``({"general_parameters": {"data_output_filename": 1.1, "data_output_path": tmpfile}}, "1.1", tmpfile)``
also, what about testing 
({"general_parameters": {"data_output_filename": 1.1, "data_output_path": desktop}}, "1.1", "~/Desktop")